src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonarcloud-github-static-resources/v1/common/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:44259/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:44259/project/issues?id=sample&pullRequest=6997&issues=AWleS5ECCti0lzGA-VX3&open=AWleS5ECCti0lzGA-VX3)

[ISSUE_KEY]: AWleS5ECCti0lzGA-VX3